site/public/index.php

```diff
@@ -143,3 +143,4 @@ function error_handler() {
 }
 
 $core->getOutput()->displayOutput();
+?>
```

Comment:
Was this change intentional? The ending tag for `php` is optional and I think in some cases of how we send the buffer of output we wouldn't want an ending tag here

Reply:
This was something I added while trying differnent approaches and I thought it might not create problem to have it there , I will correct this .